src/app/wizard/step/cluster/component.ts

```diff
@@ -296,10 +305,14 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
   }
 
   updateCNIPluginOptions() {
-    if (this.controlValue(Controls.CNIPlugin) === CNIPlugin.Cilium) {
-      this.availableProxyModes = ['ipvs', 'iptables', 'ebpf'];
+    if (
+      this.controlValue(Controls.CNIPlugin) === CNIPlugin.Cilium &&
+      !!this.isKonnectivityEnabled &&
```

Comment:
It's already a boolean. Why the double negation?

Reply:
Good catch, it's useless 👍 
Fixed